examples/kubernetes/cilium.yaml

```diff
@@ -25,6 +25,10 @@ data:
   # If you want to run cilium in debug mode change this value to true
   debug: "false"
   disable-ipv4: "false"
+
+  # Allow prometheus to scrape on this addr:port. Not specifying an address
```

Comment:
Do we want to turn it on by default? What if I, as a user, don't want it by default? Can you document it what I need to do?